notebooks/lesson08_group-project.ipynb

```diff
@@ -0,0 +1,949 @@
+{
```

Comment:
<p>instead of saying "Fit" and "Predict" would it be simpler to just use the sample submission file as follows:</p><p> </p><pre class="ql-syntax" spellcheck="false">preds = train['price'].median()

submission = pd.read_csv('sample_submission.csv')
submission['Predicted'] = preds

etc etc
</pre><p> </p><p>this would allow us to avoid the copy and rename step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I changed it a bit and used your approach to fill the submission file. However, I left the fit/predict step in because I thought it would make it clearer what they should do with the train and test set. If you think it is overkill I can delete it.</p>